### ISSUE TITLE
uBO UI isn’t accessible via TalkBack screen reader on Android

### ISSUE THREAD

**Issue by DimitrisKalaboukas** [uBlockOrigin/uBlock-issues#3223]
### Prerequisites

- [X] I verified that this is not a filter list issue. Report any issues with filter lists or broken website functionality in the [uAssets issue tracker](https://github.com/uBlockOrigin/uAssets/issues).
- [X] This is **NOT** a [_YouTube_](https://www.github.com/uBlockOrigin/uAssets/issues/20586), [_Facebook_](https://www.github.com/uBlockOrigin/uAssets/issues/3367) or [_Twitch_](https://www.github.com/uBlockOrigin/uAssets/issues/5184) report. _These sites MUST be reported by clicking their **respective links**_.
- [X] This is not a support issue or a question. For support, questions, or help, visit [/r/uBlockOrigin](https://www.reddit.com/r/uBlockOrigin/).
- [X] I performed a [cursory search of the issue tracker](https://github.com/uBlockOrigin/uBlock-issues/issues?q=is%3Aissue) to avoid opening a duplicate issue.
- [X] The issue is not present after disabling uBO in the browser.
- [X] I checked the [documentation](https://github.com/gorhill/uBlock/wiki) to understand that the issue I am reporting is not normal behavior.

### I tried to reproduce the issue when...

- [X] uBO is the only extension.
- [X] uBO uses default lists and settings.
- [X] using a new, unmodified browser profile.

### Description

uBlock Origin extension UI isn’t accessible via TalkBack screen reader on Android.

### A specific URL where the issue occurs.

```yaml
everywhere
```

### Steps to Reproduce

1. Install uBO on a compatible Android browser (e.g. Firefox Mobile)
2. Enable Google’s TalkBack accessibility app for people with visual difficulties
3. Open the main extension UI

### Expected behavior

The UI’s text and buttons are readable and double-tappable via TalkBack.

### Actual behavior

The UI sounds like blank space and isn’t functional when TalkBack is enabled.

### uBO version

1.57.2

### Browser name and version

Mozilla Firefox for Android 125.2.0

### Operating System and version

Android 14

**Comment by gwarser** [comment]
It does not work in the popup mode. Works from the logger (when you manage to turn it on), except for the Zapper, Picker and Reporter buttons. Dynamic filtering also cannot be changed. Host names are read correctly.

Scrolling the top menu in the dashboard is difficult.

Accessing the logger form Support tab in the Dashboard is difficult, because for some reason links are read separately, sentence does not start with "The logger", but reads only "is the primary tool to diagnose filter-related issues."

The Reload, DOM inspector and Toggle the popup panel buttons are not accessible, but the console button works fine. The Clear logger button is skipped - should probably be described as disabled.
